crypto/ledger/ledger_test.go

```diff
@@ -82,7 +83,7 @@ func TestPublicKeyUnsafeHDPath(t *testing.T) {
 		// Store and restore
 		serializedPk := priv.Bytes()
 		require.NotNil(t, serializedPk)
-		require.True(t, len(serializedPk) >= 50)
+		require.True(t, len(serializedPk) >= 40)
```

Comment:
switching to proto, the serialized private key seems to be shorter. Why did we need 50 before?

Reply:
Probably a prefix?